app/src/main/java/org/simple/clinic/instantsearch/InstantSearchEvent.kt

```diff
@@ -34,3 +34,8 @@ data class BlankBpPassportResultReceived(val blankBpPassportResult: BlankBpPassp
 
   override val analyticsName: String = "Instant Search : Blank BP Passport Result Received"
 }
+
+sealed class BpPassportScanned : InstantSearchEvent() {
+
+  data class ByPatientFound(val patientId: UUID) : BpPassportScanned()
```

Comment:
Why did we add a `By` here? What we actually want to portray is an `and` right? Why don't we just do `BpPassportScanned.PatientFound` instead?